src/feedback-rating/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface FeedbackRatingProp {
+    imgSrc?: string;
+    description?: string;
+    buttonLabel?: string;
```

Comment:
convention in this repo is to specify optional props as `propName?: string | undefined;`

Reply:
updated types